pubspec.yaml

```diff
@@ -47,6 +47,7 @@ flutter:
     - packages/html_editor_enhanced/assets/summernote-lite.min.css
     - packages/html_editor_enhanced/assets/summernote-lite-dark.css
     - packages/html_editor_enhanced/assets/summernote-lite.min.js
```

Comment:
remove the outdated version

Reply:
No, we want to keep it so we can back it up later in case of failure. `summernote-lite-v2.min.js` is just a modification of `summernote-lite.min.js` made by us, not from the original library of `summernote`.